### PR TITLE
[RHDX-246] User login improvements for Status Message, Expandable content

### PR DESCRIPTION
**Pull request by ghost**
### JIRA Issue Link
* https://projects.engineering.redhat.com/browse/RHDX-267

#### Expandable
1. Navigate to `/user/login` unauthenticated.
2. See that the "Legacy Login" button is no longer visible.

#### Status Message
1. Navigate to `/user/login` unauthenticated
2. Open browser developer console.
3. Use the element inspector to find the div `.layout-content .region-content #block-rhdp2-content form.user-login-form` and deselect the `display: none` property.
4. Enter incorrect username and password.
5. See updated error message using PatternFly.

### Screenshot
Screen Shot 2020-02-12 at 1 14 18 PM

### CODE REVIEW

**Comment by alistairmc** [comment]
@rogeralloymagnetic can we use the inline style alerts pf-m-inline

<div class="pf-c-alert pf-m-success pf-m-inline" aria-label="Success alert">
  <div class="pf-c-alert__icon">
    <i class="fas fa-check-circle" aria-hidden="true"></i>
  </div>
  <h4 class="pf-c-alert__title">
    <span class="pf-screen-reader">Success alert:</span>Success alert title</h4>
  <div class="pf-c-alert__description">
    <p>Success alert description.
      <a href="#">This is a link.</a>
    </p>
  </div>
  <div class="pf-c-alert__action">
    <button class="pf-c-button pf-m-plain" type="button" aria-label="Close success alert: Success alert title">
      <i class="fas fa-times" aria-hidden="true"></i>
    </button>
  </div>
</div>
<br />

**Comment by rogeralloymagnetic** [comment]
@alistairmc Sure, I've pushed an update.

**Comment by alistairmc** [comment]
@rogeralloymagnetic looks like the e2e tests are broken on this

**Comment by rogeralloymagnetic** [comment]
Yeah, @alistairmc I was worried about that. I'll take a look.

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/system/misc/status-messages.html.twig [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  {% elseif type == 'status' %}
+    {% set classes = classes|merge(['pf-m-success']) %}
+  {% elseif type == 'warning' %}
+    {% set classes = classes|merge(['pf-m-warning']) %}
+  {% endif %}
+
+  <div {{ attributes.addClass(classes)|without('role', 'aria-label') }} aria-label="{{ status_headings[type] }}" role="contentinfo">
+
+    <div class="pf-c-alert__icon">
+      {% if type == 'custom' %}
```
Comment (staceymosier) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/system/misc/status-messages.html.twig]: Can these be set by line 32, to reduce checking again?

**Review by staceymosier — CHANGES_REQUESTED** [review]

**Comment by rogeralloymagnetic** [comment]
Based off of wrong branch. Moved to: https://github.com/redhat-developer/developers.redhat.com/pull/3509

**Pull request closed without merging**